config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml

```diff
@@ -239,7 +239,7 @@ periodics:
         type: Directory
 
 - name: ci-kubernetes-e2e-kubeadm-kind-master
-  interval: 1h
+  interval: 4h
```

Comment:
my only objection would be to have this job more frequent.
if less than 4h possibly 2h.
WDYT?

Reply:
ok, 2h seems reasonable to me